src/elexmodel/models/NonparametricElectionModel.py

```diff
@@ -10,6 +10,10 @@ class NonparametricElectionModel(BaseElectionModel):
     def __init__(self, model_settings={}):
         super().__init__(model_settings)
         self.robust = model_settings.get("robust", False)
+        self.modeled_bounds_agg = None
+        self.conformalization_data_agg = None
+        self.gaussian_bounds_unit = None
```

Comment:
gaussian bounds don't make sense in this case. I think it would be better not to create the object but instead just return `None` in `get_conformalization_data_unit` and `get_conformalization_data_agg`

Reply:
changed